one_fm/hooks.py

```diff
@@ -358,8 +358,6 @@
 		"after_insert":"one_fm.tasks.erpnext.user.after_insert",
 	},
 	"Operations Post": {
-		# "after_insert":"one_fm.events.operations_post.after_insert",
-		# "on_trash": "one_fm.events.operations_post.on_trash",
 		"before_save": "one_fm.events.operations_post.before_save",
```

Comment:
Operations Post is ONEFM doctype, add before_save event in the doctype class, remove from hook.